### PR TITLE
Init project

### PR DESCRIPTION
**Pull request by Luffytaro22**
> # Setup project with components 👍 
> gif

> ## Features 🆕 
> * Setup `React` with `Vite` 🆗 
> * Use of `React Router` to render different pages 💯 
> * Created `components` for each `book state`, render `all books`, display a `form` and a `navigation` header with 2 different routes 😎 
> * Created `styles modules` for each component with the basic display 🎨

### CODE REVIEW

**Review by Strangeal — COMMENTED** [review]
Hi @Luffytaro22, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

- [ ] **It appears that your submitted project is missing this essential component. To rectify this, kindly consider the following improvements:**
  - Utilize the [README](https://github.com/microverseinc/readme-template) template provided in the project requirements, This will ensure consistency and completeness in your documentation. 
  - Customize the template to suit your project details. Provide a clear and concise overview of your project, including its key features, future features, and any relevant instructions or prerequisites for users. 
  - Consider consuming sections such as installation instructions, usage examples, configuration details, and information about any external dependencies or libraries used. Include information about the project's license, copyright, or any other legal considerations. 
  - Proofread and format your README to ensure it is easy to read, well-organized, and free of errors. Remember, documenting your project through a professional README is crucial for other developers to understand, use, and contribute to your project effectively.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/App.jsx [hunk 1]
```diff
@@ -1,34 +1,16 @@
...
-        </p>
-      </div>
-      <p className="read-the-docs">
-        Click on the Vite and React logos to learn more
-      </p>
-    </>
+    <Routes>
+      <Route path="/" element={<NavBar />}>
+        <Route index element={<Home />} />
+        <Route path="categories" />
```
Comment (franclobo) [inline src/App.jsx]: - [ ] Please 🙏🏽 correct the Route tag it should be just two routes at this moment one for the index page that corresponds to the bookstore and the other for categories, e.g. `<Route path="/" element={<Books />} />` ensure to use the path and element attributes to create the routes.

File: src/App.jsx [hunk 2]
```diff
@@ -1,34 +1,16 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
-import './App.css'
+import { Route, Routes } from 'react-router-dom';
```
Comment (franclobo) [inline src/App.jsx]: - [ ] Kindly import `BrowserRouter, Routes, Route, Link` components to avoid errors in the compiling.

File: src/Routes/Home.jsx
Comment (franclobo) [inline src/Routes/Home.jsx:1]: - [ ] The Routes directory and Home file are not part of the tree for this milestone, may we be confused about how to use routes? Please check this [article]([url](https://reactrouter.com/en/main/router-components/browser-router))

**Review by franclobo — CHANGES_REQUESTED** [review]
Hi @Luffytaro22, 

# Required Changes ♻️

so-close-jim-halpert

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

- No linter errors ✔️ 
- Descriptive PR ✔️ 
- Correct Git flow ✔️ 
- Good initializing React ✔️ 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Well done using Vite.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.jsx (hunk 1 above)
Comment (Luffytaro22) [reply src/App.jsx]: Excuse me, but I don't understand. The way I see it adds only two routes: the index and the "/categories", which are subroutes

**Review by levy002 — APPROVED** [review]
## Hello @Luffytaro22 
 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

  unnamed


## Highlights
- No linter error :heavy_check_mark: 
- You implemented all suggested changes :heavy_check_mark: 
 
## Status: APPROVED :fire:  :fire: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**